packages/react-widgets/src/NumberInput.tsx

```diff
@@ -186,9 +211,9 @@ class NumberPickerInput extends React.Component {
         className="rw-widget-input"
         onChange={this.handleChange}
         onBlur={this.handleBlur}
-        aria-valuenow={value}
-        aria-valuemin={isFinite(min) ? min : null}
-        aria-valuemax={isFinite(max) ? max : null}
+        aria-valuenow={value as any as number /*HACK*/}
```

Comment:
value should aways be a number?

Reply:
Yes: `'aria-valuenow'?: number;`

Reply:
i mean tho when can `value` itself not be a number that you need to cast to any and back to number, is this b/c of `round`?